### PR TITLE
added prices for ffood

### PR DESCRIPTION
**Pull request by tWido**
added price parsing for meals in freefood and faynfood

### CODE REVIEW

File: venza.py [hunk 1]
```diff
@@ -28,7 +28,10 @@ def __str__(self):
         return self.fmt("{name} - {place}")
 
     def __repr__(self):
-        return self.fmt("<Meal('{name}', place='{place}')>")
+        return self.fmt("<Meal('{name}',"
+                        " place='{place}'"
+                        " price='{price}'"
```
Comment (mrshu) [inline venza.py]: @tWido can you please remove the quotiation marks (`'`) from this line? Price is no longer a string

File: venza.py [hunk 2]
```diff
@@ -126,11 +129,14 @@ def ffood(which, weekday=None):
...
                        re.DOTALL)
-
+    prices = re.findall(r'class="brand">(.*?)</span>',
+                        str(lis))
+                        
     ret = []
     for i in range(0, len(meals)):
         if ((i % 2) == 0):
-            ret.append(Meal(meals[i], which))
+            price =  float(prices[i+1][:-3])
```
Comment (mrshu) [inline venza.py]: @tWido can you add a short comment on why this is necessary and why `[:-3]`?

**Comment by mrshu** [comment]
@tWido Thanks for doing this! It looks pretty good, except for those two very small nitpicks.

Please fix those and we can merge it in!

**Comment by tWido** [comment]
added suggested comment

File: venza.py [hunk 3]
```diff
@@ -28,7 +28,10 @@ def __str__(self):
         return self.fmt("{name} - {place}")
 
     def __repr__(self):
-        return self.fmt("<Meal('{name}', place='{place}')>")
+        return self.fmt("<Meal('{name}',"
+                        " place='{place}'"
+                        " price= {price}"
```
Comment (mrshu) [inline venza.py]: @tWido sorry to require you to reiterate again but can you get rid od that space after `=` on this line? Thanks!

**Comment by mrshu** [comment]
@tWido another very small comment, I think this will really be the last one.

**Pull request merged**